### PR TITLE
chore: add prettier-tailwind plugin and format all css files

### PR DESCRIPTION
**Pull request by grshane**

### CODE REVIEW

**Comment by shaal** [comment]
This PR is a great example of why using prettier with Tailwind makes no sense.

Take a look at all the files that changed. How does that help?

I would love to see prettier with regular CSS, when each css rule has its own line.
When you change margin for example, you would see the update just in margin.

**Review by sarahp — APPROVED** [review]
Looks good to me!

File: src/components/base/outline-card/outline-card.ts [hunk 1]
```diff
@@ -80,6 +80,9 @@ export class OutlineCard extends OutlineElement {
   }
 
   render(): TemplateResult {
+    console.warn('warn');
+    console.error('warn');
+    console.log('warn');
```
Comment (himerus) [inline src/components/base/outline-card/outline-card.ts]: What's up here?

**Review by himerus — CHANGES_REQUESTED** [review]
I'm curious, where do we define that we are USING the plugin? Is there no change to the tailwind config? I apologize, I didn't read the article that talked about the new feature.

**Comment by grshane** [comment]
> This plugin follows Prettier’s autoloading convention, so as long as you’ve got Prettier set up in your project, it’ll start working automatically as soon as it’s installed.

**Review by himerus — APPROVED** [review]

**Comment by himerus** [comment]
image

As a note, if you use multiple `@apply` directives to separate concerns as shown above, this plugin will only format them PER line. 🔥 🔥 🔥

**Comment by himerus** [comment]
> This PR is a great example of why using prettier with Tailwind makes no sense.
> 
> Take a look at all the files that changed. How does that help?

This helps by providing consistency with our current tools. That is all.

**Pull request merged**